src/main/java/uk/ac/kcl/Main.java

```diff
@@ -43,14 +45,25 @@ public static void main(String[] args) {
         for (File listOfFile : listOfFiles) {
             if (listOfFile.isFile()) {
                 if (listOfFile.getName().endsWith(".properties")) {
-                    System.out.println("Properties sile found:" + listOfFile.getName() +
+                    LOG.info("Properties sile found:" + listOfFile.getName() +
```

Comment:
```suggestion
                    LOG.info("Properties file found:" + listOfFile.getName() +
```

Reply:
Brilliant!
For all the time I though that the original author meant `sile` as some derivation from the word silo, hence I was not touching this one  🤣